modules/compliance-operator-uninstall.adoc

```diff
@@ -17,33 +17,14 @@ To remove the Compliance Operator, you must first delete the Compliance Operator
 
 To remove the Compliance Operator by using the {product-title} web console:
 
-. Remove CRDs that were installed by the Compliance Operator:
+. Switch to the *Administration* -> *Operators* -> *Installed Operators* page.
```

Comment:
The right procedure is:
1. first delete any scansettingbinding, complaincesuite, compliancescan
2. delete all profilebundle
3. Uninstall Operator
otherwise from GUI the operator was uninstalled. However, many resources left in the namespace, including scansettingbinding, compliancesuite, complaincescan, profilebinding, namespace.